syntax = "proto3";

service DummyService {
    rpc analyzeImage (AnalyzeImageRequest) returns (AnalyzeResponse) {};
}

message AnalyzeImageRequest {
    bytes image = 1;
}

message AnalyzeResponse {
    string description = 1;
}
